src/server/security/index.md

```diff
@@ -0,0 +1,44 @@
+# Secure Mergin Maps installation
+
+For security and privacy reasons Mergin Maps deployments should enable HTTPS secured connection via certificate file.
```

Comment:
I would add some disclaimer at beginning that we are not responsible for security breach and the approach described is just an example and need to be tweaked based on the production deployment setup by expert

Reply:
I added it [now](https://github.com/MerginMaps/docs/pull/557/commits/efe0d5f95d2de995b9d2ce4a6e8e56e4e6fc6994)